bindings/pyroot_experimental/PyROOT/test/ttree-tchain_setbranchaddress.py

```diff
@@ -0,0 +1,113 @@
+import unittest
+from array import array
+
+import ROOT
+from libcppyy import SetOwnership
```

Comment:
Does `ROOT` import `libcppyy`? Could this be available as `ROOT.SetOwnership` as well if the answer is yes?

Reply:
No, `ROOT` does not import `libcppyy` (from Python). The functionality in `libcppyy` is used in the experimental PyROOT from the pythonizations implemented in C++.
The `SetOwnership` method is used now in the tests to prevent ROOT and Python to do a double delete of objects. We basically do not want cppyy proxies to try to delete their corresponding C++ objects because ROOT's TFile will do it as well. It is a temporary workaround so that we can test the pythonizations. But the ownership of objects and interferences between ROOT C++ and Python is definitely a topic we need to tackle at some point.

Reply:
Ah, I see. This releases ownership from cppyy, not Python or ROOT. I thought this was disabling ROOT ownership of the objects.